src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -333,30 +333,45 @@ class FreezeBase : public StackObj {
 protected:
   JavaThread* const _thread;
   ContinuationWrapper& _cont;
-  CONT_JFR_ONLY(FreezeThawJfrInfo _jfr_info;)
   bool _barriers;
   const bool _preempt; // used only on the slow path
+  const intptr_t * const _frame_sp;
```

Comment:
Is this the sp of the first frame on the stack that we are going to freeze?

Reply:
Yes